packages/knip/fixtures/plugins/hardhat/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@fixtures/hardhat",
+  "dependencies": {
+    "hardhat": "*"
```

Comment:
Looks like it's typically used as dev dep? Seeing `--save-dev` in their docs.

Reply:
Yes

Reply:
Changed.  Does that make a difference for production mode?

Reply:
Indeed. In general just better to have the fixtures as close to reality as possible.